src/privileged/multipreffer/api.js

```diff
@@ -65,6 +66,17 @@ this.FirefoxHooks = {
           return;
         }
       }
+      for (const ndPref of nonDefaultPrefs) {
```

Comment:
I just realized this won't work, lol, since the expected value can be falsey. We'd need to do something like:
```
const expectValue = prefs.expectValues.hasOwnProperty(name);
if ((expectValue && Preferences.get(name) !== prefs.expectValues[name]) ||
    (!expectValue && Preferences.isSet(name))) {
  return;
}
```